src/CommandsRouter.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import Validate from "./components/Validate/Validate";
+import Help from "./components/Help/Help";
+import { CliInput } from "./CliModels";
+
+
+const commandsDictionary = (cliInput: CliInput) => ({
+  ['help']: <Help message={cliInput.helpMessage}/>,
```

Comment:
```
node dist/cli.js --help
```

☝🏼 returns help while 👇🏼 
```
node dist/cli.js help
```

returns error

```
Missing required flag
        --context, -c
```

Reply:
Solved, I think that it will manage it. I removed the component and all the related code because it doesn't work as expected 😓